tests/ingresstests/ingressclass_test.go

```diff
@@ -432,20 +479,19 @@ func TestUpdateInfraSettingInIngressClass(t *testing.T) {
 	// update infrasetting ref in ingressclass, model changes
 	g := gomega.NewGomegaWithT(t)
 
-	ingClassName, ingressName, ns, settingName1, settingName2 := "avi-lb", "foo-with-class", "default", "my-infrasetting", "my-infrasetting2"
+	ingClassName, ingressName, ns, settingName1, settingName2 := "avi-lb", "foo-with-class", "default", "my-infrasetting1", "my-infrasetting2"
```

Comment:
Same comment as above.

Reply:
No failures with the infrasetting here. My thought was to follow the same naming convention as my-infrasetting2.